ner/model.py

```diff
@@ -1,10 +1,29 @@
 import pandas as pd
 from sqlalchemy.orm import sessionmaker
+import logging
+import os
+import spacy
 
-from ner import settings
-from ner.db import DB
+#from ner import settings
+#from ner.db import DB
 
-print(settings.env)
+#print(settings.env)
+
+def extract_entities(input_text):
+    # get latest model
+    models_dir = os.path.join('..', 'Models') if __name__ == '__main__' else 'Models'
```

Comment:
we need better management around the base path. this is typically managed as an attribute in the `settings` class.